### PR TITLE
got Get with indexes to work

### PR DESCRIPTION
**Pull request by ykravtsov**

### CODE REVIEW

**Comment by CjBuresch** [comment]
Probably need to update the unit tests for RPCs and what not.

File: src/blackdoor/cqbe/rpc/Rpc.java [hunk 1]
```diff
@@ -39,6 +39,14 @@ public static Rpc fromJson(JSONObject rpcJson) throws RPCException {
...
 			case "GET":
 				rpcObject = new GetRpc();
+				GetRpc rpcGetObject = (GetRpc) rpcObject;
+				try {
+					rpcGetObject.index = rpcJson.getJSONObject("params")
+							.getInt("index");
+				}
+				catch (JSONException e){
+					throw new RPCException(JSONRPCError.INVALID_PARAMS);
+				}
```
Comment (CjBuresch) [inline src/blackdoor/cqbe/rpc/Rpc.java]: I think we're gonna want to do this kind of checking back in the Rpc class itself, in a constructor or something? @kag0 what say you?

Comment (kag0) [inline src/blackdoor/cqbe/rpc/Rpc.java]: We would prefer to do it in a protected constructor, but this will do for now.

**Comment by kag0** [comment]
So many formatting changes in here that it's really hard to find what code was changed. Try not to reformat the whole file, only change areas you are working on. This also invites massive merge conflicts.

**Pull request merged**